packages/editor/src/components/rich-text/index.native.js

```diff
@@ -92,35 +98,32 @@ export class RichText extends Component {
 	 * handler.
 	 *
 	 */
-	splitContent( htmlText, start, end ) {
+	splitContent( currentRecord, blocks = [], isPasted = false ) {
```

Comment:
> isPasted = false

I understand that we just need this flag here but, may I suggest copying what the web side is providing the corresponding method with? The idea is to stay as close as possible to the web code, wishing we can refactor and re-use the same code eventually.

I this case, let's pass a `context` object with a `paste` (boolean) field. WDYT?